Autopilot/Src/freertos.c

```diff
@@ -118,6 +124,18 @@ void StartDefaultTask(void const * argument)
   /* USER CODE END StartDefaultTask */
 }
 
+/* Interchip_Run function */
+__weak void Interchip_Run(void const * argument)
```

Comment:
This is definitely not the correct way to do this. **Declare** the function in `Interchip_A.h` (note that it must take a `void const *` parameter than can be casted any way you like), then call it from here. **OR** `#include "Interchip_A.h` at the top of this file and write the loop here. You shouldn't be writing `__weak` functions outside of a library context.

Reply:
I used the cube autogeneration for this, I can define it as weak, external, or normal. I'll fix the function parameters (I'm confused why it works without the correct signature, but it does). I'll change it to not use weak if that isn't conventional.